### ISSUE TITLE
The image at (https://fluidproject.org/projects.html) featured demos get disappeared when color and contrast in changed.

### ISSUE THREAD

**Issue by Chaitanyassr** [fluid-project/fluidproject.org#29]
The image at (https://fluidproject.org/projects.html) featured demos get disappeared when color and contrast in changed.

### To reproduce
1. Go to https://fluidproject.org/projects.html by clicking on project on the main site.
2. Click on preference
3. change the color and contrast.

## Screenshots
![Screenshot (169)](https://user-images.githubusercontent.com/30374226/75628676-3b6ddd80-5c01-11ea-9735-5f8ee0168ef9.png)

 - Browser [e.g. chrome]

**Comment by jobara** [comment]
These images are likely supplied as background images. UIO contrast options remove backgrounds images by default. Do we want to have keep these background images, remove them, keep them but change their colours, something else?

**Comment by Chaitanyassr** [comment]
> These images are likely supplied as background images. UIO contrast options remove backgrounds images by default. Do we want to have keep these background images, remove them, keep them but change their colours, something else?

We can keep those images or we can also do something like github do for pinned repo, something very similer to this. chainging the color should not be an option as it would disrupt the look and feel.
![Screenshot (171)](https://user-images.githubusercontent.com/30374226/75801224-c76a3b80-5da0-11ea-9aa1-ca401d1afb50.png)

**Comment by jobara** [comment]
@Chaitanyassr can you explain in more detail the suggestion that is similar to pinned repos on GitHub?

**Comment by sachin10101998** [comment]
@jobara it's better if we keep these images even when contrast or theme is changed. Colour and Contrast options menu states that it is used to change text and background color. It's supposed to remove background colors and not images.

**Comment by sachin10101998** [comment]
This issue is similar to #28

**Comment by Chaitanyassr** [comment]
> @Chaitanyassr can you explain in more detail the suggestion that is similar to pinned repos on GitHub?

As we can see in pinned section of github overview, repos(those we pinned) are organized inside the boxes. Its look quite aesthetic. We can do this for the featured demos(https://fluidproject.org/projects.html) were featured demos would be inside the same type of boxes.this would solve issue #28 #29.

**Comment by sachin10101998** [comment]
@Chaitanyassr The background images disappear on changing themes because the background property explicitly sets the background to none. It's not because they are not inside the same types of boxes.
> We can do this for the featured demos(https://fluidproject.org/projects.html) were featured demos would be inside the same type of boxes.this would solve issue #28 #29.

Can you elaborate on how putting those featured demos inside the same types of boxes will solve #28 and @29?

**Comment by Chaitanyassr** [comment]
> @Chaitanyassr The background images disappear on changing themes because the background property explicitly sets the background to none. It's not because they are not inside the same types of boxes.
> 
> > We can do this for the featured demos(https://fluidproject.org/projects.html) were featured demos would be inside the same type of boxes.this would solve issue #28 #29.
> 
> Can you elaborate on how putting those featured demos inside the same types of boxes will solve #28 and @29?

true, i totally agreed with the fact that background image can be kept. as @jobara asked of what we can do other that, so we can make a featured demo pannel like the one used for github pinned repo. Its not about the issue, its about replacing the whole featured demo thing while keeping #28 and #29 in mind.

**Comment by sachin10101998** [comment]
@Chaitanyassr Correct me if I'm wrong but are you saying we remove the background images from default contrast theme too?
